src/Compilers/CSharp/Portable/CodeGen/EmitAddress.cs

```diff
@@ -68,7 +68,6 @@ private LocalDefinition EmitAddress(BoundExpression expression, AddressKind addr
 
                     if (expression.Type.IsValueType)
                     {
-
                         if (!HasHome(expression, addressKind))
```

Comment:
>if (!HasHome(expression, addressKind)) [](http://example.com/codeflow?start=24&length=38)

It doesn't look like there are any meaningful changes in this file. Consider reverting.